src/components/YourCompany/CompanyName/CorrectCompanyName.vue

```diff
@@ -81,10 +81,3 @@ export default class CorrectCompanyName extends Mixins(CommonMixin) {
   }
 }
 </script>
-
-<style lang="scss" scoped>
-::v-deep #company-name-input {
-  // hide uppercase transformation delay from user
-  text-transform: uppercase;
-}
-</style>
```

Comment:
why are we removing this?

Reply:
Its purpose was to show in all-caps whatever the user entered. It is no longer used here.